### PR TITLE
Add requirements.txt and update .gitignore

### PR DESCRIPTION
**Pull request by racehd**
- Added ability to capture 1-2 word Lore DC's and sluggify them into an inline check. 
- Added a requirements.txt for easier local set-up
- Updated .gitignore with a template since my .venv wasn't ignored previously.

### CODE REVIEW

File: dataEntry.py [hunk 1]
```diff
@@ -231,6 +231,14 @@ def handle_inlines_checks(string):
     string = sub(r"basic %s save against your class DC" % SAVES, r"@Check[type:\1|dc:resolve(@actor.system.attributes.classDC.value)|basic:true] save against your class DC", string)
     string = sub(r"%s save against your class DC" % SAVES, r"@Check[type:\1|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC", string)
     
+    # Lores - capture 1-2 words between 'DC ##' and 'Lore'
+    def sluggify_lore(match):
```
Comment (TikaelSol) [inline dataEntry.py:235]: We actually don't need to sluggify lores, `@Check[type:Astronomy Lore]` works automatically and doesn't need a localized label. Unless that broke, In which case we should fix that as a bug.

We'd also want to move the function definition out away from the main method.

Comment (racehd) [reply dataEntry.py:235]: Hadn't realized non-sluggified worked now. Will update!

**Comment by TikaelSol** [comment]
I'm integrating the lore changes locally. It is maybe obvious that the exe version is several versions past the py version and that's going to continue until August 1st for reasons.

Can you pull the lore changes out and just have this be the gitignore and requirements changes?

**Comment by racehd** [comment]
Sure thing, i reverted dataentry.py to master

**Pull request merged**